.github/workflows/build-and-deploy-to-ck8s.yaml

```
@@ -7,11 +7,10 @@ on:
       - demo-*
 
 env:
-  DOMAIN: ckdemo.a1ck.io
-  APP_DOMAIN: ${{ github.ref_name }}.ckdemo.a1ck.io
+  DOMAIN: CHANGE_ME
+  APP_DOMAIN: ${{ github.ref_name }}.CHANGE_ME
   DOCKER_USER: 'robot$demo+ci-cd'
   REGISTRY_PROJECT: demo
```

I'm confused. Why not simply:

```suggestion
  REGISTRY_PROJECT: demo
  TAG: ${{ github.ref_name }}-${{ github.sha }}
```

Getting philosophical, but I'd expect either:

```
TAG=v1.0.1 # for a tag
TAG=branch_name-random_crap_like_job_id_or_sha # for a branch
```

---

Great suggestion, have gone with this approach!